### DISCUSSION TITLE
Possibillity to make that players only can build in their plot/area

### DISCUSSION THREAD

**Discussion by raamford** [GriefPrevention/GriefPrevention#1573] [Q&A]
hey is there a way that you can only allow players to build in the world on their plot or on which they are trustet? It would be very helpful!

**Comment by Jikoo** [comment]
Yes - you want either `Creative` or `SurvivalRequiringClaims` [claim mode](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on).
